pages/system/checkboxes.js

```diff
@@ -28,54 +28,85 @@ export default class SystemPageCheckboxes extends React.Component {
         <br />
         <br />
         <br />
-        <System.H2>Usage</System.H2>
+        <System.H2>Imports</System.H2>
         <hr />
         <br />
         <System.P>
-          Define the boolean Checkbox states and handle the state change the when user checks or unchecks the CheckBox.
+          Import React and the CheckBox Component.
         </System.P>
         <br />
+        <br />
         <System.CodeBlock>
-          {`state = {
-  six: false,
-  seven: true,
-};
-
-_handleChange = (e) => {
-  this.setState({ [e.target.name]: e.target.value });
-}; `}
+{`import * as React from 'react';
+import { CheckBox } from 'slate-react-system';`}
         </System.CodeBlock>
         <br />
         <br />
+        <System.H2>Usage</System.H2>
+        <hr />
+        <br />
         <System.P>Declare the CheckBox component.</System.P>
         <br />
 
         <System.CodeBlock>
-          {`<System.CheckBox name="six" value={this.state.six} onChange={this._handleChange}>
-  <strong>Unchecked</strong>
-  <br />
-  This CheckBox default is unchecked.
-</System.CheckBox>
+{`class ExampleOne extends React.Component {
+   state = { exampleOne: false }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`